datadog_cluster_agent/assets/configuration/spec.yaml

```
@@ -0,0 +1,12 @@
+name: Datadog-Cluster-Agent
+files:
+- name: datadog_cluster_agent.yaml
+  options:
+  - template: init_config
+    options:
+      - template: init_config/openmetrics
```

```suggestion
      - template: init_config/openmetrics_legacy
```
Needs resync

`init_config/openmetrics` template refers to options that correspond with OpenmetricsV2BaseCheck

---

You're right, unfortunately I realized that I was using the wrong template after submitting the PR and seeing that the config github check failed.

@clamoriniere and @hithwen proposed to make this integration work with `OpenMetricsV2BaseCheck` and fallback to `OpenMetricsBaseCheck` when running on Python 2 (V2 only works with python 3). The idea is to implement something similar to what you already have in the linkerd integration.

Do you think we should merge this one first with `OpenMetricsBaseCheck` and `init_config/openmetrics_legacy` and then do a separate PR with V2 and the fallback or should I just try to do everything in this one?